client/src/components/ImageGallery.jsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import {Grid, Col, Row} from 'react-styled-flexboxgrid';
+
+const WrapperOuter = styled.div`
+  display: block;
+  box-sizing: border-box;
+`;
+
+const WrapperInner = styled.div`
+  align-items: stretch;
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: center;
+  box-sizing: border-box;
+  width: 1140px;
+  margin: auto;
+  position: relative;
+`;
+
+const Box1 = styled.div`
+  flex: 2;
+`;
+
+const Box23 = styled.div`
```

Comment:
For lines 26 - 52, these styled-components all seem to be doing the same thing. Would it be possible to combine these into one rule and then use that one throughout the project?

If it is more of a readability matter, where it is easier to tell which section you are in when in the render portion, I understand that as well.

Reply:
Yes. That is true. Thanks. This, I can refactor it now. I have just made a commit.